src/util/RouterUtil.js

```diff
@@ -124,7 +124,7 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
 
       var successData = {
         user: res._embedded.user,
-        type: res.type
+        type: res.type || Enums.SESSION_SSO
```

Comment:
theoretically this changed the logic.
before: type is always either STEP_UP or SESSION_SSO.
after: type is STEP_UP or SESSION_SSO or anything when `res.type` is not null.

Am curious what else the `res.type` could be beside STEP_UP and SESSION_SSO from API.
This will change the expectation in case developers rely on the type.

Reply:
You right, was thinking under the assumption that the comment `// Add the type for now until the API returns it.` means that the type was `null` (except for `SESSION_STEP_UP`) and not something else.

Reply:
would be better to get confirmed regarding than merely trust comments

Reply:
it can be `SESSION_STEP_UP` or `CREDENTIAL_VERIFY` i believe. Probably better to avoid changing behavior for now. It's a bit of a shame because we don't really use the value correctly or effectively, but we can worry about that later.

Reply:
We are adding new `type` values for IdX though:

* `LOGIN`
* `ENROLL_NEW_USER`
* `RECOVER_LOGIN`